presto-main/src/main/java/io/prestosql/security/AccessControlConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.security;
+
+import io.airlift.configuration.Config;
+
+import java.io.File;
+
+public class AccessControlConfig
+{
+    private File accessControlFile;
+
+    public File getAccessControlFile()
+    {
+        return accessControlFile;
+    }
+
+    @Config("access-control")
```

Comment:
`access-control.file`  or `access-control.config` or `access-control.config-file` ?

Reply:
I like `access-control.config-file` as it's similar to `catalog.config-dir`